### DISCUSSION TITLE
Can I tell whether a file has sliced i/p/b frames?

### DISCUSSION THREAD

**Discussion by derekbillingsley** [essential61/mp4analyser#30] [Q&A]
Hi there, am I able to pull out details on whether an I-frame is sliced?  For example with another tool I can visually see that the I-frame consists of 4 separate slices (same with P and B)  Thanks!

**Comment by essential61** [comment]
mp4Analyser is a very small project (basically me doing this as a hobby with a couple of contributions from others) so it's focus is quite narrow which is to provide a visual representation of the mp4 container format. People are welcome to fork the repo and either add functionality to decode the mpeg stream syntax or possibly invoke a low-level tool like ffmpeg with context-specific parameters, say to look at an individual frame in more detail.
